python/aixexplainer/aixserver/model.py

```diff
@@ -102,6 +110,16 @@ def explain(self, request: Dict) -> Dict:
                     "masks": masks,
                     "top_labels": np.array(explanation.top_labels).astype(np.int32).tolist()
                 }}
+            elif str.lower(self.explainer_type) == "limetexts":
```

Comment:
Can we add an e2e test for limetexts like the [limeimage test](https://github.com/kserve/kserve/blob/master/test/e2e/explainer/test_aix_explainer.py)?